include/rocksdb/flush_manager.h

```diff
@@ -0,0 +1,33 @@
+//  Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
+//  This source code is licensed under both the GPLv2 (found in the
+//  COPYING file in the root directory) and Apache 2.0 License
+//  (found in the LICENSE.Apache file in the root directory).
+
+#pragma once
+
+#include <functional>
+#include <vector>
+
+namespace rocksdb {
+
+class FlushManager {
+ public:
+  virtual ~FlushManager() {}
+
+  std::function<void(uint32_t, std::vector<uint32_t>*)> OnManualFlush1;
```

Comment:
OnManualFlush1 => OnManualFlushv1. Earlier I thought it was the number of arguments. I guess you are doing this for backward compatibilty if you had to change the interface. Is that true, then in that case will you calling both the apis if there is a breaking change. Not sure how to solve this one.

Reply:
Here we have two functions, internal one i.e. `OnManualFlush(ColumnFamilySet*, autovector<ColumnFamilyData*>*)` and external one `OnManualFlush1(uint32_t, vector<uint32_t>*)`. I expect the internal one to be much more stable than the external one.